packages/flutter/lib/src/widgets/page_storage.dart

```diff
@@ -129,7 +111,121 @@ class PageStorageBucket {
   }
 }
 
-/// A widget that establishes a page storage bucket for this widget subtree.
+/// Establish a subtree in which widgets can opt into persisting states after
+/// being destroyed.
+/// 
+/// [PageStorage] is used to save and restore values that can outlive the widget
+/// For example, when multiple pages are grouped in tabs, when a page is
+/// switched out, its widget is destroyed and its state is lost. By adding a
+/// [PageStorage] at the root and adding a [PageStorageKey] to each page, the
+/// page's state will be stored in its closest ancestor [PageStorage], and
+/// restored when it's switched back.
+/// Usually you don't need to explicitly use a [PageStorage], since it's already
+/// included in routes.
+/// [PageStorageKey] is used by [Scrollable] if
```

Comment:
Either add an empty line before this line, or format it into a block. A single newline isn't displayed in doc.